src/dialog.scss

```diff
@@ -31,6 +31,8 @@ $button: #{$fd-namespace}-button;
   $fd-dialog-content-box-shadow: var(--sapContent_Shadow3);
   $fd-dialog-content-border-radius: var(--sapElement_BorderCornerRadius);
   $fd-dialog-content-padding-x: (def: 1rem, s: 1rem, m: 2rem, l: 2rem, xl: 3rem);
+  $fd-dialog-content-min-width-x: (def: 0, s: 0, m: 600px, l: 1024px, xl: 1440px);
+  $fd-dialog-content-max-width-x: (def: 599px, s: 599px, m: 1023px, l: 1439px, xl: 0);
```

Comment:
This is causing some display issues in the docs - I tried to adjust them a bit but any further input on how to better display them would be appreciated.